### ISSUE TITLE
safespoting? or something not tolerated?

### ISSUE THREAD

**Issue by SuperHeroesAlwaysdie** [LightsHope/server#842]
![1](https://user-images.githubusercontent.com/33297493/33349679-7b88a6d8-d49b-11e7-9e09-1a8035f97fdd.PNG)

![2](https://user-images.githubusercontent.com/33297493/33349693-891d607c-d49b-11e7-8477-173b4a346608.PNG)

picture and title are quite explicite.

I didn't do GM request for this because of the rapidity and the utility(of doing a request) they have currently so i post it here.

**Comment by Ravyd** [comment]
Yeah alliance has been shooting through walls for months, it is long past time this got patched. Before patch 1.7 the Grom Gol flight master was dead at least 8/10 times I visited there.  The only reason he isn't camped now is that the population of GG has increased dramatically. But still, 2/10 times he is still dead.

**Comment by ChazyTheBest** [comment]
It seems it's a LoS problem, some gob trigger it, but the map and part of the map gob don't (like shooting / casting through elevated terrain and walls, etc. I wonder why devs don't pick a more up to date cmangos code (or even switching to tc, downgrading can be heavy but it's definitely worth it...).

**Comment by SuperHeroesAlwaysdie** [comment]
just like in av near the horde final boss, alliance is offen using this bug as well, and this is well known... i will take screen of this when i will have the opportinity

**Comment by thepparker** [comment]
Casting through various terrain and many game objects (such as trees, smaller fences, lamp posts, etc) are all true to Vanilla. The walls in Grom'Gol, however, probably should be blocking LoS.
